privacyidea/webui/login.py

```diff
@@ -134,6 +134,17 @@ def single_page_application():
     else:
         customization_baseline_file = "templates/baseline.html"
 
+    login_text = policy_object.get_action_values(
+        allow_white_space_in_action=True,
+        action=ACTION.LOGIN_TEXT,
+        scope=SCOPE.WEBUI,
+        client=client_ip, unique=True
```

Comment:
Hm, this is a general problem with ``single_page_application``: We don't use ``get_client_ip`` here. Thus, one can just pass a custom ``X-Forwarded-For`` header to spoof the ``client_ip`` variable here. Do you think we should prevent this from happening?

Reply:
Hm, interesting. Why didn't we use get_client_ip in the first place?
(I know, because we did only allow the X-Forwarded-For for /validate/ requests till now).

So if we would use ``get_client_ip`` in line 84, then the admin would have to allow the proxy to overwrite the client.
I think this would be ok. 
However, would change the behaviour - worth noting in the changelog...

Reply:
done

Reply:
:+1: